engine/pkg/cmd/cli/cli_job_create.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+	"io"
+	"os"
+
+	"github.com/pingcap/log"
+	"github.com/pingcap/tiflow/engine/enginepb"
+	engineModel "github.com/pingcap/tiflow/engine/model"
+	cmdcontext "github.com/pingcap/tiflow/pkg/cmd/context"
+	"github.com/pingcap/tiflow/pkg/errors"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+// createJobOptions defines flags for job create.
+type createJobOptions struct {
```

Comment:
`createJobOptions` and `jobGeneralOptions` should be named in the same style, maybe `jobCreateOptions ` is appropriate?